### ISSUE TITLE
OperationBase is not available in new versions of relay-runtime

### ISSUE THREAD

**Issue by ericls** [relay-tools/relay-hooks#48]
It only exists in very old version of relay-runtime. 
With newer versions the compiler throws:
```
@types/relay-runtime has no exported member 'OperationBase'
```

**Comment by morrys** [comment]
Hi @ericls,
Thanks for the report :)

With this change https://github.com/relay-tools/relay-hooks/commit/0500bf11a9e5fb92c6b4baf59ffc80b1dc719acd the problem should be solved. These days I will make further checks in order to release the change over the weekend.

**Comment by morrys** [comment]
hi @ericls,
today I released version 2.1.0 where I solved this issue:
https://github.com/relay-tools/relay-hooks/releases/tag/v2.1.0

For any problem, open a new issue.
thanks
